Fuser/auto_agent.py

```diff
@@ -325,6 +330,7 @@ def __init__(
         verify: bool = True,
         dispatch_jobs: int = 2,
         allow_fallback: bool = True,
+        target_platform: Optional[PlatformConfig] = None,
```

Comment:
```suggestion
        target_platform: Optional[str] = None,
```

Is this ever passed in as a Config object?